### PR TITLE
UX: Make youtube embeds full width by default

### PR DESCRIPTION
**Pull request by xfalcox**
Before

image

After
image

### CODE REVIEW

**Comment by SamSaffron** [comment]
Is this going to work on mobile?

**Comment by xfalcox** [comment]
> Is this going to work on mobile?

It does work @SamSaffron, but the ratio gets busted.

I pushed a new one, that while more complex, makes it fluid and more adaptable.

File: plugins/lazy-yt/plugin.rb [hunk 1]
```diff
@@ -35,11 +38,15 @@ def to_html
...
-           data-height="#{video_height}"
+           #{size_restricted ? size_tags.map { |t| "data-#{t}" }.join(' ') : ""}
            data-parameters="#{embed_params}">
         <a href="https://www.youtube.com/watch?v=#{video_id}" target="_blank">
-          <img class="ytp-thumbnail-image" src="#{thumbnail_url}" width="#{video_width}" height="#{video_height}" title="#{escaped_title}">
+          <img class="ytp-thumbnail-image"
+               src="#{thumbnail_url}"
+               #{size_restricted ? size_tags.join(' ') : ""}
+               width="#{video_width}"
+               height="#{video_height}"
```
Comment (davidtaylorhq) [inline plugins/lazy-yt/plugin.rb:48]: If size_restricted is true, I think this will leave us with duplicate width/height attributes?

**Comment by eviltrout** [comment]
How risky is this change? Should it be moved post release?

File: plugins/lazy-yt/plugin.rb (hunk 1 above)
Comment (xfalcox) [reply plugins/lazy-yt/plugin.rb]: Oops, forgot to delete those lines. Thanks for the catch!

Fixed in 8a68db4

**Comment by xfalcox** [comment]
> How risky is this change? Should it be moved post release?

I'd say it is safe enough to make the cut.

**Pull request merged**